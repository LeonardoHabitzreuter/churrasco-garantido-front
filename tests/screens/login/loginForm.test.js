import React from 'react'
import { shallow } from 'enzyme'
import LoginForm from '../../../src/screens/login/loginForm'

describe('Login form component', () => {
  test('should renders properly', () => {
    const wrapper = shallow(<LoginForm />)
    expect(wrapper).toMatchSnapshot()
  })
})
