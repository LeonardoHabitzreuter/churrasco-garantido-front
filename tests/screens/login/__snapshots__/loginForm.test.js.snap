// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login form component should renders properly 1`] = `
<div>
  <Alert
    className="alert"
    handleDismiss={[Function]}
    messages={Array []}
    show={false}
    style="danger"
  />
  <Form
    fields={
      Array [
        Object {
          "labelName": "Email",
          "name": "email",
          "required": true,
          "type": "email",
          "value": "",
        },
        Object {
          "labelName": "Senha",
          "maxLength": 20,
          "minLength": 6,
          "name": "password",
          "required": true,
          "type": "password",
          "value": "",
        },
      ]
    }
    formGroupsClass="formGroup"
    onSubmit={[Function]}
  />
</div>
`;
